pages/landing/hipaa/technical-details/index.html

```diff
@@ -53,6 +55,239 @@
     Request early access to receive exclusive benefits and discounts!
     <ul class="check-list check-white" style="text-align: left; display: inline-block;"><li><span class="help-text light" title="" data-toggle="tooltip" data-original-title="You’ll be the first to gain access to the solution once it’s ready"> Pre-release HIPAA-Compliant Infrastructure </span> </li><li><span class="help-text light" title="" data-toggle="tooltip" data-original-title="Early adopters will receive the solution at a major discount!"> Early adopter discount</span> </li><li><span class="help-text light" title="" data-toggle="tooltip" data-original-title="We’ll work with you to ensure the solution meets your organization’s requirements"> Influence the product direction</span> </li><li><span class="help-text light" title="" data-toggle="tooltip" data-original-title="We’ll go through the audit with you, implementing any technical changes indicated by the auditor"> HITECH audit support</span> </li></ul>
   cta: Request early access
+
+infra_how_it_works:
+  - heading: End-to-end, opinionated architecture for AWS
+    image: /assets/img/hipaa/infrastructure-hipaa.png
+    image_side: right
+    body: |
+      The Reference Architecture is an opinionated, battle-tested, best-practices way to assemble the code from the <a href="/infrastructure-as-code-library"/>Infrastructure as Code Library</a>) into an end-to-end tech stack that includes just about everything you need: server cluster, load balancer, database, cache, network topology, monitoring, alerting, CI/CD, secrets management, VPN, and more (check out the <a href="/devops-checklist/">Production Readiness Checklist</a> to see what it takes to go to prod).
+
+  - heading: Customizable
+    image: /assets/img/hipaa/hipaa-sample-app.png
+    image_side: left
+    body: |
+      You can choose from EKS, ECS or EC2, databases such as MySQL, Postgres, SQL Server, to customize the Reference Architecture to your exact specifications.
+
+  - heading: Fully defined as code using Terraform, Docker, Packer
+    tab_code_preview: site.data.hipaa-landing-page-tabs-ref-arch-code
+    image_side: right
+    body: |
+      You keep and can modify 100% of the code to support any use case you might have.
+
+  - heading: Is HIPAA-compliant out of the box
+    image: /assets/img/hipaa/grunty-audit.png
+    image_width: 350px
+    image_side: left
+    body: |
+      We'll have our infrastructure and applications certified by a third party auditor to ensure HIPAA compliance, and pass all the best practices on to you.
+
+  - heading: Gruntwork deploys it into your AWS accounts and gives you all of the code to manage it, in about 1 day
+    image: /assets/img/hipaa/stay-up-to-date.png
+    image_width: 500px
+    image_side: right
+    body: |
+      Instead of spending months (or years) to re-invent the wheel, let us handle the Gruntwork for you, so you can focus on your core deliverables
+
+categories:
```

Comment:
This lite/pro/enterprise thing was for Heroku. HIPAA requires a different table with different features. From my original comment on this:

> Perhaps we have a similar table for HIPAA, except the columns are “Gruntwork Standard,” “Gruntwork CIS,” and “Gruntwork HIPAA” and we highlight a bunch of the HIPAA requirements as rows you only get with the HIPAA subscription: e.g., CIS hardened base images, anti-virus, file integrity monitoring, etc. We would also X out any services not eligible for HIPAA (if any).

Reply:
+1 on the quoted suggestion. I believe we actually had a table very much along those lines at one point, so we can probably pull it back in and add some additional rows to flesh out the distinctions more clearly.

Reply:
Roger :+1:

Reply:
Table implemented in 3afc7ed5. Took a first pass at the copy, but this still needs some additional HIPAA infra requirements - and some of the items that originated from the index page's table should be dropped - but the styling and structure is in place now.